about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>About CodeCamp</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+        <script src="https://kit.fontawesome.com/f41acb8d77.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="./fonts/cocogoose.css" />
+        <link rel="stylesheet" href="./css/styles.css" />
+        <link rel="stylesheet" href="./css/about.css" />
+        <link rel="stylesheet" href="./css/menu.css" />
+    </head>
+    <body>
+        <main>
+            <nav class="toggle-nav">
+                <button type="button" class="close-menu"><i class="fa fa-times fa-spin" aria-hidden="true"></i></button>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>CodeCamp Campaign</li>
+                </ul>   
+            </nav>
```

Comment:
- [ ] Good job implementing the navbar in your mobile version, but currently I can't navigate to the `home page` using the navbar menu as you can see in the screenshot below, Kindly fix this issue

![navbar_issue](https://user-images.githubusercontent.com/80895497/158706800-c15ace37-7bb3-4773-8767-94c129d953d5.png)